doc/architecture/adr/0020-authentication-orm.md

```diff
@@ -0,0 +1,33 @@
+# Authentication ORM
+
+* Status: Accepted
+* Deciders: @MortMH, @Frydensberg1, @CodeReaper, @robertrosborg
+* Date: 2023-02-06
+
+---
+
+## Context and Problem Statement
+
+We need an ORM for the Auth domain to handle entity mapping and database migrations.
+
+---
+
+## Considered Options
+
+* Entity Framework
+* Dapper
+
+---
+
+## Decision Outcome
+
+We have chosen Entity Framework.
+
+## Rationale
+
+Both Entity Framework and Dapper are capable ORMs, but we've chosen Entity Framework for the following main reasons:
+
```

Comment:
If we found out anything else interesting during testing and discussing, we should try to mention it here... My memory is not the best tho.